packages/client/src/components/Navigation/Navbar.component.js

```diff
@@ -34,7 +39,8 @@ export const Navbar = ({ ...props }) => {
           {accountStatus}
           <div className="favourite-container">
             <Link
-              className={`navbar-link ${!user ? 'disabled-link' : ''}`}
+              className={`navbar-link `}
```

Comment:
You can simplify this to `className="navbar-link"` without the template literals and brackets. They seem to be a leftover from before.